sandbox/.gitignore

```diff
@@ -1,4 +1,4 @@
-**/*
+/*
 !README.md
```

Comment:
Didn’t realize this package has its own `.gitignore`. Can we move this to the top level as we’ve done with all the other gitignore files.

Reply:
Moved